### PR TITLE
LEV-292: added manage organisation page 

### PR DESCRIPTION
**Pull request by nBaqai**
- added manage organisation list when access /admin/organisations
- redirect to organisation if user accessing only managing 1 organisation
- fixed issue when clicking on the GOV.UK to go back to main page was throwing error now fixed

### CODE REVIEW

File: controllers/UserManagement/ManageOrgsController.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      const orgsResult = await orgLookup({
+        ...this.getOptions(req),
+        url: '/admin/organisations'
+      });
+
+      const orgs = orgsResult.organisations;
+
+      if (orgs.length == 1) {
+        return _res.redirect(`/admin/organisations/${orgs[0].id}`);
+      } else {
```
Comment (ashley-bickerstaff-homeoffice) [inline controllers/UserManagement/ManageOrgsController.js:16]: You can reduce the cyclomatic complexity here by removing the `else` as the first conditional block `return`s the function:

```
if (orgs.length == 1) {
    return _res.redirect(`/admin/organisations/${orgs[0].id}`);
}
req.sessionModel.set('orgsResult', orgs);
next();
```

File: controllers/UserManagement/ManageOrgsController.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const BaseController = require('../BaseController');
+const { orgLookup } = require('../../services/UserManagement/OrganisationSearchService');
+
+class ManageOrgsController extends BaseController {
```
Comment (ashley-bickerstaff-homeoffice) [inline controllers/UserManagement/ManageOrgsController.js:4]: I think this should be called "OrganisationsController". I appreciate it's close to "OrganisationController" but the singular vs plural tells us something. Perhaps we could add a comment indicating what this page is doing:

```suggestion
// Page that lists multiple organisation or redirects if there is only one
class OrganisationsController extends BaseController {
```

File: locales/en/default.json [hunk 3]
```diff
@@ -164,6 +164,10 @@
       "organisationPage": {
         "title": "Manage organisation",
         "pageTitle": "Organisation"
+      },
+      "manageOrganisationPage": {
```
Comment (ashley-bickerstaff-homeoffice) [inline locales/en/default.json:168]: - [ ] Just a note that should change too if we go with the rename

File: views/pages/organisation/manage.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{% extends "app-template.njk" %}
```
Comment (ashley-bickerstaff-homeoffice) [inline views/pages/organisation/manage.html:1]: - [ ] Just a note that should change too if we go with the rename

**Review by ashley-bickerstaff-homeoffice — CHANGES_REQUESTED** [review]
Just a couple of changes here

File: controllers/UserManagement/ManageOrgsController.js (hunk 1 above)
Comment (nBaqai) [reply controllers/UserManagement/ManageOrgsController.js]: Done the changes

**Pull request merged**